test/cloudformation/iam_cloudformation.yaml

```diff
@@ -86,6 +86,7 @@ Resources:
             Resource:
               - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:instance-profile/KarpenterNodeInstanceProfile-*"
               - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/KarpenterNodeRole-*"
+              - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:instance-profile/instance-profile-drift"
```

Comment:
Can you add a comment here saying what this is explicitly used for in the future, so that people know what's needed and what isn't?